packages/flutter/lib/src/gestures/pointer_router.dart

```diff
@@ -149,4 +152,12 @@ class FlutterErrorDetailsForPointerRouter extends FlutterErrorDetails {
 
   /// The pointer event that was being routed when the exception was raised.
   final PointerEvent event;
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
```

Comment:
I'm also happy just removing FlutterErrorDetails and adding these properties into FlutterError objects generated in the 1 or 2 places this is used. Note that these 3 useful properties were currently falling on the ground and not exposed to anyone because it was previously hard to expose them without breaking the foundation, render, widgets layering.